pkgs/applications/editors/emacs/elisp-packages/manual-packages/lspce/module.nix

```diff
@@ -34,6 +34,6 @@ rustPlatform.buildRustPackage {
     homepage = "https://github.com/zbelial/lspce";
     description = "LSP Client for Emacs implemented as a module using Rust";
     license = lib.licenses.gpl3Only;
-    maintainers = with lib.maintainers; [ AndersonTorres ];
+    maintainers = with lib.maintainers; [ ];
```

Comment:
There is an update script in `package.nix`.  Could you remove that?

Reply:
Done!